scripts/main/lychee.js

```diff
@@ -358,6 +358,22 @@ lychee.load = function(autoplay = true) {
 			lychee.footer_hide();
 
 
+		} else if (albumID=='search') {
```

Comment:
This does not work correctly for me when the search bar at the top is empty and I'm activating it to type a search query. With your change, I'm being immediately redirected to an empty album, probably because `photoID` is empty?

Reply:
should be fixed